src/main/java/io/cdap/delta/bigquery/BigQueryUtils.java

```diff
@@ -179,6 +179,21 @@ private static long executeAggregateQuery(BigQuery bigQuery, String query, Encry
     return val.getLongValue();
   }
 
+  /**
+   * Get normalized dataset name for target dataset
+   * Method returns normalized datasetName if it is not empty
+   * Otherwise it returns normalized form of databaseName
+   * @param datasetName dataset name to be normalized
+   * @param databaseName database name to be used if datasetName is empty
+   * @return normalized datasetName if it is not empty otherwise normalized form of databaseName
+   */
+  public static String getNormalizedDatasetName(@Nullable String datasetName, String databaseName) {
+    if (StringUtils.isNotEmpty(datasetName)) {
```

Comment:
we can use `Strings.IsNullOrEmtpy()` method from guava instead of introducing new dependency for this case.